### PR TITLE
fix/VONK-6560-envvar-syntax

### PR DESCRIPTION
**Pull request by cknaap**

### CODE REVIEW

**Comment by louisledev** [comment]
@cknaap  I switched to draft because it includes doc for `member-match` operation which should not be part of 5.7.

File: setting_up_firely_server/configuration/environment_variables.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ The format for the environment variables is:
 
     VONK_<setting_level_1>[:<setting_level_n>]*
 
-So you start the variable name with the prefix 'VONK\_', and then follow the properties in the json settings, separating each level with a colon ':'. Some examples:
+So you start the variable name with the prefix ``VONK_``, and then follow the properties in the json settings, separating each level with a colon ``:``. Some examples:
```
Comment (louisledev) [inline setting_up_firely_server/configuration/environment_variables.rst:28]: Not you code, but we should replace the `:` separator by the `__` as this is more widely supported. see https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-8.0#non-prefixed-environment-variables

File: setting_up_firely_server/configuration/environment_variables.rst [hunk 2]
```diff
@@ -51,11 +51,6 @@ environment variable::
 
 	VONK_Administration:SqlDbOptions:ConnectionString=<some connectionstring>
 
-To access an array item, use 0-based indexing::
-
-	VONK_PipelineOptions:Branches:0:Exclude:0=Vonk.Repository.Memory
-	VONK_PipelineOptions:Branches:0:Exclude:1=Vonk.Repository.Sql
-
```
[the remark this answers is not in the document]
Comment (cknaap) [reply setting_up_firely_server/configuration/environment_variables.rst:58]: This page is about explaining how to translate settings to environment variables. And the lines above show how to do that for array indexes. However, that is done in a separate paragraph right below as well. Hence I removed this superfluous example.

**Comment by cknaap** [comment]
Because this PR was accidentally on top of another change (member-match), I close it in favor of #449

**Pull request closed without merging**